src/transceiver.c

```diff
@@ -612,20 +670,27 @@ uint8_t wait_for_trans_cmd_resp(uint8_t expected_len) {
 
 bool send_trans_cmd(uint8_t expected_len, char* fmt, ...) {
     va_list args;
+    va_start(args, fmt);
+    vsnprintf((char *) command_buf, PRINT_BUF_SIZE, fmt, args);
```

Comment:
Shouldn't be `PRINT_BUF_SIZE`, should be either `COMMAND_BUF_SIZE` or `sizeof(command_buf)`